src/EditorFeatures/TestUtilities/TestExportProvider.cs

```diff
@@ -123,6 +124,7 @@ private static ComposableCatalog CreateAssemblyCatalogWithCSharpAndVisualBasic()
         {
             return ExportProviderCache
                 .GetOrCreateAssemblyCatalog(GetCSharpAndVisualBasicAssemblies(), ExportProviderCache.CreateResolver())
+                .WithoutPartsOfType(typeof(CommitConnectionListener))
```

Comment:
:bulb: Would like to see a future PR remove this exclusion, but this probably keeps the current PR clean.